app/Views/admin/categories/Categories.php

```diff
@@ -1 +1,265 @@
-Categories
\ No newline at end of file
+<?= $this->extend('templates/admin/dashboardTable') ?>
+
+<?= $this->section("title-meta") ?>
+<?php echo view('partials/title-meta', array('title' => 'Categoria')); ?>
+<?= $this->endSection() ?>
+
+<?= $this->section("css") ?>
+<link href="https://cdn.datatables.net/1.13.6/css/dataTables.bootstrap5.min.css" rel="stylesheet">
+<link href="https://cdn.datatables.net/buttons/2.4.2/css/buttons.bootstrap5.min.css" rel="stylesheet">
+<link href="https://cdn.datatables.net/responsive/2.2.9/css/responsive.bootstrap5.min.css" rel="stylesheet">
+<?= $this->endSection() ?>
+
+<?= $this->section("content") ?>
+<div class="page-content">
+    <div class="container-fluid">
+
+        <?php echo view('partials/page-title', array('pagetitle' => 'Pages', 'title' => 'Categorías')); ?>
+
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="card">
+                    <div class="card-header d-flex justify-content-between align-items-center flex-wrap">
+                        <h5 class="card-title mb-2 mb-md-0">Tabla de Categorías</h5>
+                        <div class="mt-2 mt-md-0 float-md-right">
+                            <button type="button" class="btn btn-success btn-label waves-effect waves-light rounded-pill">
+                                <i class="ri-add-circle-fill label-icon align-middle rounded-pill fs-16 me-2"></i>Agregar Categoría
+                            </button>
+                        </div>
+                    </div>
+                    <div class="card-body">
+                        <table id="category" class="table nowrap dt-responsive align-middle table-hover table-bordered" style="width:100%">
```

Comment:
```suggestion
                        <table id="categories-table" class="table nowrap dt-responsive align-middle table-hover table-bordered" style="width:100%">
```
change the name of the id